### ISSUE TITLE
Doesnt work at all

### ISSUE THREAD

**Issue by Eevori** [Crimix/FriendlyGriefing#2]
I added this mod to ATM 8 and maybe its a ATM 8 overall issue with many other mods in the pack but changing the whitelist does nothing, for example changing it to minecraft:creeper and then restarting the server ignore the list and defaults back to villager so regardless of the list it just uses villager

**Comment by Crimix** [comment]
Have you tried editing the config while the server is not running? I have also experience something like that from other mods

**Comment by Eevori** [comment]
yes i restarted the server before editing and also after, no changes, and if this mod does work as standalone then it must be a modpack issue
